src/app/index.html

```diff
@@ -84,6 +84,7 @@
 	<script src="lib/models/generic_collection.js"></script>
 	<script src="lib/models/movie_collection.js"></script>
 	<script src="lib/models/stream_info.js"></script>
+	<script src="lib/models/webtorrent_stream_info.js"></script>
```

Comment:
webtorrent_stream_info.js its not completed, since i couldn't get the same info that peerflix supply.
I need help in this one.